cmd/utils/flags.go

```diff
@@ -1000,10 +1000,12 @@ func setNodeUserIdent(ctx *cli.Context, cfg *node.Config) {
 // setBootstrapNodes creates a list of bootstrap nodes from the command line
 // flags, reverting to pre-configured ones if none have been specified.
 func setBootstrapNodes(ctx *cli.Context, cfg *p2p.Config) {
-	urls := params.MainnetBootnodes
+	urls := params.EtherumMainnetBootnodes
 	switch {
 	case ctx.GlobalIsSet(BootnodesFlag.Name):
 		urls = SplitAndTrim(ctx.GlobalString(BootnodesFlag.Name))
+	case ctx.GlobalBool(MainnetFlag.Name):
```

Comment:
We never set this flag when running, so this looks incorrect. The default must be `RoninMainnetBootnodes` and in this case when mainnet flag is set `urls = params.EtherumMainnetBootnodes`

Reply:
Yes, checked this flag, it's used for Etherum Mainnet. I have the otherway for creating flag for Ronin but it requires more flag and updating in entrypoint.sh, so let go with this suggestion.